src/aui/framemanager.cpp

```diff
@@ -1154,7 +1156,9 @@ wxString wxAuiManager::SavePaneInfo(const wxAuiPaneInfo& pane)
     result += wxString::Format(wxT("floatx=%d;"), pane.floating_pos.x);
     result += wxString::Format(wxT("floaty=%d;"), pane.floating_pos.y);
     result += wxString::Format(wxT("floatw=%d;"), pane.floating_size.x);
-    result += wxString::Format(wxT("floath=%d"), pane.floating_size.y);
+    result += wxString::Format(wxT("floath=%d;"), pane.floating_size.y);
+    result += wxString::Format(wxT("cli_floatw=%d;"), pane.floating_client_size.x);
+    result += wxString::Format(wxT("cli_floath=%d"), pane.floating_client_size.y);
```

Comment:
I wouldn't bother extending the old serialization format, but if we do, we probably need to change the layout version, i.e. use `layout3` instead of `layout2`.